piicatcher/__init__.py

```diff
@@ -1,5 +1,6 @@
 # flake8: noqa
 __version__ = "0.20.2"
+__google_analytics_tid__ = "UA-148590293-1"
```

Comment:
Ideally this should not be added to code or the repository. However, I don't have a better idea for the usage here.